locale/de/LC_MESSAGES/django.po

```diff
@@ -17,58 +17,58 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 
-#: bundesliga_app/forms.py:17
+#: bundesliga_app/forms.py:26
 msgid "Insert a name code"
 msgstr "Einen Namen eingeben"
 
-#: bundesliga_app/forms.py:23
+#: bundesliga_app/forms.py:32
 msgid "Insert your discount"
 msgstr "Fügen Sie Ihren Rabatt ein"
 
-#: bundesliga_app/forms.py:39 templates/buyer/get_discount.html:97
+#: bundesliga_app/forms.py:59
+msgid "You cant create a discount in a free event"
+msgstr "Sie können in einer kostenlosen Veranstaltung keinen Rabatt erstellen"
+
+#: bundesliga_app/forms.py:69
+msgid "You already have a discount for this ticket type"
+msgstr "Login um die Rabatte Ihrer Events zu verwalten"
+
+#: bundesliga_app/forms.py:101
 msgid "Insert your member number here"
 msgstr "Geben Sie hier Ihre Mitgliedsnummer ein"
 
-#: bundesliga_app/forms.py:59
+#: bundesliga_app/forms.py:122
 msgid "Invalid request"
 msgstr "Ungültige Anfrage"
 
-#: bundesliga_app/forms.py:75
+#: bundesliga_app/forms.py:138
 msgid "Invalid number "
 msgstr "Ungültige Nummer "
 
-#: bundesliga_app/forms.py:80
+#: bundesliga_app/forms.py:143
 msgid "Invalid numbers "
 msgstr "Ungültige Nummern "
 
-#: bundesliga_app/forms.py:88
+#: bundesliga_app/forms.py:151
 msgid "You already used the discount for this event"
```

Comment:
Is this when a member try to use their number in the same event twice?

if it that case, probably would be better IMO "You already used your membership to obtain a discount in this event"